pre_commit/clientlib.py

```diff
@@ -156,6 +156,30 @@ def check(self, dct: Dict[str, Any]) -> None:
             )
 
 
+class OptionalSensibleSlashesAtHook(cfgv.OptionalNoDefault):
+    def check(self, dct: Dict[str, Any]) -> None:
+        super().check(dct)
+
+        if r'[\/]' in dct.get(self.key, ''):
+            logger.warning(
+                f'Pre-commit normalizes the slashes in {self.key!r} field in '
```

Comment:
pre-commit is not capitalized

Reply:
I capitalized it as a sentence, even though it wasn't one. Fixed.